src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -166,6 +173,12 @@ if (ENABLE_MLAS_FOR_CPU)
     target_include_directories(${TARGET_NAME} SYSTEM PRIVATE $<TARGET_PROPERTY:mlas,INCLUDE_DIRECTORIES>)
     add_definitions(-DOV_CPU_WITH_MLAS)
 endif()
+
+if(ENABLE_SHL_FOR_CPU)
+    target_link_libraries(${TARGET_NAME} PRIVATE shl)
+    target_include_directories(${TARGET_NAME} SYSTEM PRIVATE $<TARGET_PROPERTY:shl,INCLUDE_DIRECTORIES>)
```

Comment:
can we add public properties for `shl` target? In this `target_link_libraries` with `shl` is enough without explicit `target_include_directories`

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/23901/commits/02e86443a35737f168c75c62885a1873353c08be. Could you please review one more time? Thanks!